tests/app/submitter/test_convert_payload_0_0_3.py

```diff
@@ -1351,3 +1351,79 @@ def test_relationship_answers_not_on_path_in_payload(version):
     assert ("related-to-anyone-else-answer", "person1") in answers
     relationships_answer = answers[("relationship-answer", None)]
     assert expected_relationships_answer == relationships_answer["value"]
+
+
+@pytest.mark.parametrize(
```

Comment:
I would add a test to ensure the `answer_codes` property isn't in the payload when no answers